### PR TITLE
[15.0][MIG] email_from

### PR DESCRIPTION
**Pull request by nikunjantala**

### CODE REVIEW

**Comment by manengel** [comment]
@nikunjantala @jans23 I had a very short look on that module and, well: The module is confusing me. On Odoo such things are normally handed on a per-template basis. You can go to Settings -> Technical -> [in section E-Mail] to Templates and change things like the (default) sender, the model.

So....is this module really a benefit you knowingly want to have or did you just bypass the standard of Odoo without being aware of it?

**Comment by jans23** [comment]
The idea is to use this module once instead of changing each template.

**Comment by manengel** [comment]
@jans23 Ah O.K., I will then do the review as requested.

File: email_from/views/res_config_settings_views.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                                    <tree editable="bottom">
+                                        <field name="email_from" string="Email From" />
+                                        <field
+                    name="from_email_model_ids"
+                    string="Models"
+                    widget="many2many_tags"
+                  />
+                                    </tree>
+                                </field>
+                            </div>
```
Comment (manengel) [inline email_from/views/res_config_settings_views.xml:39]: The formatting looks really strange to me.

File: email_from/views/email_from.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                    <group>
+                        <field name="email_from" required="True" />
+                    </group>
+                    <group>
+                        <field
+              name="from_email_model_ids"
+              widget="many2many_tags"
+              required="True"
+            />
+                    </group>
```
Comment (manengel) [inline email_from/views/email_from.xml:29]: Formatting.

**Review by manengel — CHANGES_REQUESTED** [review]
I commented a few small things. Just let a formatter do the job (and correct by hand if it really does not want to correct it) and you are good to go! It was (unnecessarily) hard to test this, but I will report about that at a better suited place. The module itself works technically fine!

File: email_from/views/res_config_settings_views.xml (hunk 1 above)
Comment (nileshsheliya) [reply email_from/views/res_config_settings_views.xml:39]: Its auto formatting by pre-commit command. If we not format it and push it git, pipeline will failed.

Comment (manengel) [reply email_from/views/res_config_settings_views.xml:39]: O.K. then I did not say a thing.

**Review by manengel — APPROVED** [review]

**Pull request merged**